cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -76,16 +83,7 @@ private CruiseControlConfiguration(String configuration, List<String> forbiddenO
         super(configuration, forbiddenOptions);
     }
 
-    /**
-     * Returns a CruiseControlConfiguration created without forbidden option filtering.
-     * @param string A string representation of the Properties
-     * @return The CruiseControlConfiguration
-     */
-    public static CruiseControlConfiguration unvalidated(String string) {
-        return new CruiseControlConfiguration(string, emptyList());
-    }
-
     public static Map<String, String> getCruiseControlDefaultPropertiesMap() {
-        return CC_DEFAULT_PROPERTIES_MAP;
+        return CRUISE_CONTROL_DEFAULT_PROPERTIES_MAP;
```

Comment:
Do we perhaps want to clone the map to stop any objects from modifying the defaults? (This might not be neccessary though...)

Reply:
I converted it to an unmodifiable map in the definition at the top of the file (using `CRUISE_CONTROL_DEFAULT_PROPERTIES_MAP = Collections.unmodifiableMap(config);`) this matches what we do with the zookeeper default config.

Reply:
@tomncooper remember to run checkstyle and mainly spotbugs. I had a problem with it. thanks.